client/html/templates/checkout/standard/process-body-standard.php

```diff
@@ -189,7 +193,7 @@
 			<a class="btn btn-default btn-lg" href="<?= $enc->attr( $this->standardUrlPayment ); ?>">
 				<?= $enc->html( $this->translate( 'client', 'Change payment' ), $enc::TRUST ); ?>
 			</a>
-			<button class="btn btn-primary btn-lg btn-action">
+			<button class="btn btn-primary btn-lg btn-action" id="purchase-order-btn">
```

Comment:
Do you use that ID somewhere?

Reply:
It is neccessary to catch form sumbit to JS because of we need to reorder this button from submit to send data to Stripe firstly:

$('body').on('click', '#purchase-order-btn', function (e) {
			e.preventDefault();
			AimeosPurchaseHandler.detectCurrentPaymentMethod();
			AimeosPurchaseHandler.handlePurchase();
		});

Reply:
OK, understood. Can you rename it to "payment-button" to keep it a bit shorter and clear?

Reply:
Done